reference/api/health_api.md

```diff
@@ -0,0 +1,34 @@
+# Dapr health API reference
+
+Dapr provides health checking probes that can be used as readiness or liveness of Dapr.
+
+### Get Dapr health state
+
+Gets the health state for Dapr.
+
+#### HTTP Request
+
+```http
+GET http://localhost:<daprPort>/v1.0/healthz
+```
+
+#### HTTP Response Codes
+
+Code | Description
+---- | -----------
+200  | health is ok
+500  | health is not ok
+
+#### URL Parameters
+
+Parameter | Description
+--------- | -----------
+daprPort | The Dapr port.
+
+#### Examples
+
+```shell
+curl http://localhost:3500/v1.0/healthz
+  -H "Content-Type: application/json"
+```
```

Comment:
200 OK is supposed to have request body. I wonder if we need to return simple body. e.g.

```
{"status":"ok"}
```

@yaron2 @amanbha how do you think ?

Reply:
We need to remove the header with the content type.  We don't have to return a response, a status code with no body is valid, this is what Kubernetes health checks expect also. They only examine the status code.

Reply:
Thanks, removed the header with the content type.